proposed-decisions.md

```diff
@@ -0,0 +1,128 @@
+# (PROPOSED) DECISIONS
+
+## Service credits still necessary in ecosystem?
+[Related Issue 9](https://github.com/stichtingsem/functional-overview/issues/9)
+
+Service credits will **not** be necessary in the new ecosystem. The functionality of service credits will be replaced by entitlements. If there is **no** entitlement or the quantity of the entitlement is **not** sufficient there will be **no** access for the user.
+
```

Comment:
In the context of a user, it's important to note that when there aren't enough entitlements left, an administrator can resolve that within 10 minutes through the proper channels: order additional entitlements on MP, .. etc

Reply:
You are right, but it is good to make a clear statement that there has to be an entitlement to get access. It is the responsibility of the school and market place to arrange this in time